ibis/backends/tests/test_client.py

```diff
@@ -216,7 +217,7 @@ def test_insert_no_overwrite_from_dataframe(
     con, test_employee_schema, test_employee_data_2
 ):
 
-    temp_table = 'temp_to_table'
+    temp_table = f'temp_to_table_{guid()}'
```

Comment:
These changes fix a race condition when running tests in parallel. Without this multiple clients try to remove the `temp_to_table` independently.

Reply:
I'll move this to another patch.

Reply:
#2925